Tests/SentryTests/SentrySpanContextTests.swift

```diff
@@ -62,6 +62,17 @@ class SentrySpanContextTests: XCTestCase {
         XCTAssertEqual(data["status"] as? String, "ok")
     }
     
+    func testSerialization_NotSettingProperties_PropertiesNotSerialized() {
+        let spanContext = SpanContext(operation: someOperation)
+        
+        let data = spanContext.serialize()
+        
+        XCTAssertNil(data["description"])
+        XCTAssertNil(data["sampled"])
```

Comment:
Isn't this required value?

Reply:
No, it's also nullable in java, see https://github.com/getsentry/sentry-java/blob/ee459986fe336dc1e0a11fdb0bf472ec90cb2de4/sentry/src/main/java/io/sentry/SpanContext.java#L27.